Gemfile

```diff
@@ -26,16 +24,15 @@ gem 'bower-rails'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 # Adds HTML templates into Angular's $templateCache using asset pipeline.
-gem 'angular-rails-templates'
-# NOTE: angular-rails-templates not yet compatible with sprockets > 3.0
+# angular-rails-templates > 1.0.0 now compatible with sprockets > 3.0
 # https://github.com/pitr/angular-rails-templates/issues/93
-gem 'sprockets', '~> 2.12.4'
```

Comment:
@chrisdolendo it's not technically "removing" sprockets -- sprockets is a dependency of rails so you don't normally need to specify it in the gemfile. But we had listed it in the gemfile in order to specifically tell it to use `~> 2.12.4`, however with the upgrade to `angular-rails-templates` we can just allow it to be included by default and in the `Gemfile.lock` it's using sprockets 3.7.0 (which is now compatible)